Gruntfile.js

```diff
@@ -87,4 +87,16 @@ module.exports = function(grunt) {
   // Default task.
   grunt.registerTask('default', ['test']);
 
+
+  // Support running a single test suite
+  grunt.registerTask('nodeunit:only', 'Run a single test specified by a target; usage: "grunt nodeunit:only:<module-name>[.js]"', function(file) {
```

Comment:
Oh thank god. I was just editing my grunt file to only run one file with `grunt nodeunit`. This is much better.